src/docs/arch-glossary.dox

```diff
@@ -103,8 +103,8 @@ they are used in other settings.
 <tr><td>table<td>schema<td>an abstraction that allows one or more btrees to be used in a coordinated way, which is specified by the schema.  The table coordinates indices and column groups.
 <tr><td>thread<td>general<td>a thread of control.  WiredTiger uses pthreads for POSIX systems.
 <tr><td>thread group<td>general<td>a group of threads that perform a common task.  Eviction is performed by a thread group.
-<tr><td>time window<td>durable history<td>a set of start and stop transaction ids and timestamps which determine when an update is visible.
 <tr><td>timestamp<td>transactions<td>refers to one of several types of 64 bit values used to control transactional visibility of data in WiredTiger.  See oldest timestamp, stable timestamp, commit timestamp, read timestamp.
+<tr><td>time window<td>durable history<td>a set of start and stop transaction ids and timestamps which determine when an update is visible.
```

Comment:
I will create a ticket for this, it should not be fixed here.

Reply:
Thanks, that would be great.
It's worth noting in the PR build there's another related error (s_all does not like the change made here?)
```
[2023/10/06 01:25:39.588] ####################### MESSAGE ############################
[2023/10/06 01:25:39.588] + echo s_all run of: "./s_docs" resulted in:
[2023/10/06 01:25:39.588] s_all run of: "./s_docs" resulted in:
[2023/10/06 01:25:39.588] + sed -e s/^/    / __s_all_tmp_s_docs
[2023/10/06 01:25:39.589]     89d88
[2023/10/06 01:25:39.589]     < timestamp
[2023/10/06 01:25:39.589]     90a90
[2023/10/06 01:25:39.589]     > timestamp
[2023/10/06 01:25:39.589]     =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[2023/10/06 01:25:39.589]     arch-glossary.dox entries out of order
[2023/10/06 01:25:39.589]     =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
```

Reply:
Reverting the line-shifting change made the above error go away. It's not yet clear unfortunately what could have caused the difference between my local dev env and Evergreen (both running sort version 8.30).